internal/expr/info.go

```diff
@@ -4,9 +4,13 @@ import (
 	"reflect"
 )
 
+type Outputs struct {
+	structFields []field
+}
+
 // field represents reflection information about a field from some struct type.
 type field struct {
-	fieldType reflect.Type
+	typ reflect.Type
```

Comment:
I've just realised that we don't even need to know the field type. In previous versions of Scan I had used it but now if there is a problem with the type returned from the database not matching the field type it will be caught by `Rows.Scan` rather than us. I've removed the field.